nixos/modules/system/boot/loader/grub/install-grub.pl

```diff
@@ -299,12 +299,16 @@ sub GrubFs {
         copy $font, "$bootPath/converted-font.pf2" or die "cannot copy $font to $bootPath\n";
     }
     if ($splashImage) {
-        # FIXME: GRUB 1.97 doesn't resize the background image if it
```

Comment:
Before anyone asks, this comment has been "wrong" for a while.

GRUB 1.97 may still not resize the background image, but GRUB 2.02 does!